great-idea/css/index.css

```diff
@@ -63,4 +63,201 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+* {
+	box-sizing: border-box;
+}
+.massContainer {
+	display: flex;
+	flex-direction: column;
+	margin: 0 15% 0 15%;
+
+	
+}
+
+hr {
+	color: grey;
+	width: 100%;
+	size: 5px;
+}
+
+.navContainer {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	width: 70%;
+	height: 60px;
+	margin-bottom: 30px;
+	position: fixed;
+	background: lavender;
+	border: 2px black;
+	border-style: none solid solid none;
```

Comment:
I really love the style of your nav, the border on the bottom and the right looks super slick. The lavender is a wonderful choice as well.